crypto/src/main/java/net/corda/v5/crypto/CordaOID.java

```diff
@@ -7,9 +7,7 @@
  * If an OID is incorrectly assigned, it should be marked deprecated and NEVER be reused again.
  */
 public final class CordaOID {
-    private CordaOID() {
-        // this class is never constructed; it exists for the static strings only
```

Comment:
I don't agree with removing this, I know we talked about it before and people felt it was a standard pattern, but it is better to be explicit.

However I won't block the PR for it,  it's a pretty minor matter.

Reply:
Yes good point, it's just that's how we have it elsewhere in the API code base in similar cases (i.e. for not constructable classes) so I believe we should be aligned, either remove these or add it everywhere.